### ISSUE TITLE
Pippin v2.1.0 connection refused issue?

### ISSUE THREAD

**Issue by qwahzi** [appditto/pippin_nano_wallet#28]
Just updated to v2.1.0 and I'm struggling to troubleshoot this new issue:

```
# curl -d '{"action": "block_count"}' 127.0.0.1:7076
{
    "count": "163924163",
    "unchecked": "11",
    "cemented": "163924163",
    "full": "30366437",
    "pruned": "133557726"
}

# curl -d '{"action": "block_count"}' 127.0.0.1:11338
curl: (7) Failed to connect to 127.0.0.1 port 11338: Connection refused
```

```
# systemctl status pippin
● pippin.service - Pippin Wallet
   Loaded: loaded (/etc/systemd/system/pippin.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2022-08-24 12:13:19 CDT; 8min ago
 Main PID: 2545 (pippin-server)
    Tasks: 1 (limit: 22773)
   Memory: 41.6M
   CGroup: /system.slice/pippin.service
           └─2545 /usr/local/bin/python3.9 /usr/local/bin/pippin-server

Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:     tr, pr = await compat.wait_for(connector, timeout=timeout)
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:   File "/usr/local/lib/python3.9/site-packages/asyncpg/compat.py", line 66, in wait_for
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:     return await asyncio.wait_for(fut, timeout)
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:   File "/usr/local/lib/python3.9/asyncio/tasks.py", line 481, in wait_for
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:     return fut.result()
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:   File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 692, in _create_ssl_connection
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:     tr, pr = await loop.create_connection(
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:   File "uvloop/loop.pyx", line 2024, in create_connection
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]:   File "uvloop/loop.pyx", line 2001, in uvloop.loop.Loop.create_connection
Aug 24 12:13:21 FreeNanoFaucet pippin-server[2545]: ConnectionRefusedError: [Errno 111] Connection refused
```

```
# pippin-server
INFO:aiohttp.server:Initializing database
INFO:aiohttp.server:💥 Using BoomPOW For Work Generation
INFO:aiohttp.server:Pippin server starting at 127.0.0.1:11338
INFO:tortoise:Tortoise-ORM started, {'default': <tortoise.backends.asyncpg.client.AsyncpgDBClient object at 0x7fecc4b5a430>}, {'db': {'Wallet': <class 'pippin.db.models.wallet.Wallet'>, 'Account': <class 'pippin.db.models.account.Account'>, 'AdHocAccount': <class 'pippin.db.models.adhoc_account.AdHocAccount'>, 'Block': <class 'pippin.db.models.block.Block'>}}
INFO:tortoise:Tortoise-ORM generating schema
ERROR:asyncio:Task exception was never retrieved
future: <Task finished name='Task-5' coro=<PippinServer.start() done, defined at /usr/local/lib/python3.9/site-packages/pippin/server/pippin_server.py:1017> exception=ConnectionRefusedError(111, 'Connection refused')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/pippin/server/pippin_server.py", line 1021, in start
    await runner.setup()
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_runner.py", line 279, in setup
    self._server = await self._make_server()
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_runner.py", line 375, in _make_server
    await self._app.startup()
  File "/usr/local/lib/python3.9/site-packages/aiohttp/web_app.py", line 417, in startup
    await self.on_startup.send(self)
  File "/usr/local/lib/python3.9/site-packages/aiosignal/__init__.py", line 36, in send
    await receiver(*args, **kwargs)  # type: ignore
  File "/usr/local/lib/python3.9/site-packages/tortoise/contrib/aiohttp/__init__.py", line 85, in init_orm
    await Tortoise.generate_schemas()
  File "/usr/local/lib/python3.9/site-packages/tortoise/__init__.py", line 636, in generate_schemas
    await generate_schema_for_client(connection, safe)
  File "/usr/local/lib/python3.9/site-packages/tortoise/utils.py", line 31, in generate_schema_for_client
    await generator.generate_from_string(schema)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/base/schema_generator.py", line 448, in generate_from_string
    await self.client.execute_script(creation_string)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/base_postgres/client.py", line 34, in _translate_exceptions
    return await self._translate_exceptions(func, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/asyncpg/client.py", line 82, in _translate_exceptions
    return await func(self, *args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/base_postgres/client.py", line 150, in execute_script
    async with self.acquire_connection() as connection:
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/base/client.py", line 326, in __aenter__
    await self.ensure_connection()
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/base/client.py", line 322, in ensure_connection
    await self.client.create_connection(with_db=True)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/asyncpg/client.py", line 59, in create_connection
    self._pool = await self.create_pool(password=self.password, **self._template)
  File "/usr/local/lib/python3.9/site-packages/tortoise/backends/asyncpg/client.py", line 65, in create_pool
    return await asyncpg.create_pool(None, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/asyncpg/pool.py", line 413, in _async__init__
    await self._initialize()
  File "/usr/local/lib/python3.9/site-packages/asyncpg/pool.py", line 441, in _initialize
    await first_ch.connect()
  File "/usr/local/lib/python3.9/site-packages/asyncpg/pool.py", line 133, in connect
    self._con = await self._pool._get_new_connection()
  File "/usr/local/lib/python3.9/site-packages/asyncpg/pool.py", line 511, in _get_new_connection
    con = await connection.connect(
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connection.py", line 2093, in connect
    return await connect_utils._connect(
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 903, in _connect
    raise last_error
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 889, in _connect
    return await _connect_addr(
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 781, in _connect_addr
    return await __connect_addr(params, timeout, True, *args)
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 833, in __connect_addr
    tr, pr = await compat.wait_for(connector, timeout=timeout)
  File "/usr/local/lib/python3.9/site-packages/asyncpg/compat.py", line 66, in wait_for
    return await asyncio.wait_for(fut, timeout)
  File "/usr/local/lib/python3.9/asyncio/tasks.py", line 481, in wait_for
    return fut.result()
  File "/usr/local/lib/python3.9/site-packages/asyncpg/connect_utils.py", line 692, in _create_ssl_connection
    tr, pr = await loop.create_connection(
  File "uvloop/loop.pyx", line 2024, in create_connection
  File "uvloop/loop.pyx", line 2001, in uvloop.loop.Loop.create_connection
ConnectionRefusedError: [Errno 111] Connection refused
```

**Comment by bbedward** [comment]
This is an unable to connect to postgres error. Can you verify postgres is running and accepting connections? If everything is good, check the postgres logs it may give some more details.

**Comment by qwahzi** [comment]
I never configured Postgres for Pippin, just the default SQLite implementation, so let me try installing/configuring Postgres to see if it resolves the issue. Thanks!

**Comment by bbedward** [comment]
@qwahzi oh, I see, I made a mistake in my implementation (forgot about SQLite)

I'll push a 2.1.1 to fix it.

**Comment by bbedward** [comment]
Fixed in a3added1efd8b42e7501d365feea376bc818efe8

**Comment by qwahzi** [comment]
Confirmed fix! Thanks again!
